File: SofiForce.Web/ClientApp/src/app/core/Models/SearchModels/ClientSurveySearchModel.ts
import { BaseSearchModel } from "./baseSearchModel";

export interface ClientSurveySearchModel extends BaseSearchModel {
    branchId: number | null;
    branchCode: string;

    clientTypeId: number | null;
    surveyId: number;
    surveyCode:string;
    serveyStatusId: number | null;
    clientId: number | null;
    clientCode:string;
    createDate: Date | null;
    startDate: Date | null;
    serveyGroupId: number | null;
    representativeId: number | null;
    representativeCode:string;
    isClosed: number | null;
}

File: SofiForce.Models/Models/SearchModels/ClientSurveySearchModel.cs
using System;
using System.Collections.Generic;

namespace Models
{
    public class ClientSurveySearchModel : BaseSearchModel
    {
        public int? BranchId { get; set; }
        public int? ClientTypeId { get; set; }
        public int SurveyId { get; set; }
        public int? ServeyStatusId { get; set; }
        public int? ClientId { get; set; }
        public DateTime? CreateDate { get; set; }
        public DateTime? StartDate { get; set; }

        public int? ServeyGroupId { get; set; }
        public int? RepresentativeId { get; set; }
        public int? IsClosed { get; set; }


    }
}